### DISCUSSION TITLE
Im not the owner of my own server (no other admin)

### DISCUSSION THREAD

**Discussion by Traineeidiot** [Pryaxis/TShock#2187] [Q&A]
Please help, as my name suggests, Im an idiot.
Ive basically locked myself out.
What can I do?
Thanks.

**Comment by Kojirremer** [comment]
If you have access to your console, then you can type there
`user group <YourUserName> owner`
to change your user to the Owner group. If your user name has spaces, then you have to wrap it between quotation marks.
